src/assets/custom/css/_sass/_soft-mod.scss

```diff
@@ -53,14 +62,6 @@
   }
 }
 
-.soft-mod__curated-publications {
```

Comment:
It's becoming less clear what relates to soft-mod and what relates to service-line pages in general. Let's look at this together

Reply:
This has been fixed in this [pull request](https://github.com/codurance/site/pull/1821)